src/detect-mark.c

```diff
@@ -253,81 +253,77 @@ static int DetectMarkPacket(DetectEngineThreadCtx *det_ctx, Packet *p,
 /**
  * \test MarkTestParse01 is a test for a valid mark value
  *
- *  \retval 1 on succces
- *  \retval 0 on failure
+ * \FAIL/PASS API
+ *  \PASS macro returns 1
+ *  \FAIL macro returns 0
  */
 static int MarkTestParse01 (void)
 {
     DetectMarkData *data;
 
     data = DetectMarkParse("1/1");
 
-    if (data == NULL) {
-        return 0;
-    }
+    FAIL_IF_NULL(data);
 
     DetectMarkDataFree(NULL, data);
-    return 1;
+    PASS;
 }
 
 /**
  * \test MarkTestParse02 is a test for an invalid mark value
  *
- *  \retval 1 on succces
- *  \retval 0 on failure
+ * \FAIL/PASS API
+ *  \PASS macro returns 1
+ *  \FAIL macro returns 0
  */
 static int MarkTestParse02 (void)
 {
     DetectMarkData *data;
 
     data = DetectMarkParse("4");
 
-    if (data == NULL) {
-        return 1;
-    }
+    PASS_IF(data == NULL);
 
     DetectMarkDataFree(NULL, data);
-    return 0;
+    FAIL;
 }
 
 /**
  * \test MarkTestParse03 is a test for a valid mark value
  *
- *  \retval 1 on succces
- *  \retval 0 on failure
+ * \FAIL/PASS API
+ *  \PASS macro returns 1
+ *  \FAIL macro returns 0
  */
 static int MarkTestParse03 (void)
 {
     DetectMarkData *data;
 
     data = DetectMarkParse("0x10/0xff");
 
-    if (data == NULL) {
-        return 0;
-    }
+    FAIL_IF_NULL(data);
 
     DetectMarkDataFree(NULL, data);
-    return 1;
+    PASS;
 }
 
 /**
  * \test MarkTestParse04 is a test for a invalid mark value
  *
- *  \retval 1 on succces
- *  \retval 0 on failure
+ * \FAIL/PASS API
```

Comment:
these values are self explaining and don't need per function documentation

Reply:
Sure!
Shall I commit the changes here, or give a separate pull request?

Reply:
Our workflow is to create a new branch + new PR for each "cycle" of the review process. So please create a new branch, new PR, then add a reference to the new PR in this old PR and close this one.